lib/model.js

```diff
@@ -670,8 +670,8 @@ class Model {
    * @param {Boolean}                 [options.omitNull] Don't persist null values. This means that all columns with null values will not be saved
    * @param {Boolean}                 [options.timestamps=true] Adds createdAt and updatedAt timestamps to the model.
    * @param {Boolean}                 [options.paranoid=false] Calling `destroy` will not delete the model, but instead set a `deletedAt` timestamp if this is true. Needs `timestamps=true` to work
-   * @param {Boolean}                 [options.underscored=false] Converts all camelCased columns to underscored if true
-   * @param {Boolean}                 [options.underscoredAll=false] Converts camelCased model names to underscored table names if true
+   * @param {Boolean}                 [options.underscored=false] Converts all camelCased columns to underscored if true. Will not convert column name if it set explicitly through options
```

Comment:
`field` is an attribute option which can override column name. Used something like this

```js
orgName: {
   type: Sequelize.STRING,
   field: 'org_name'
}
```

You are talking about timestamp fields, you can change message to something like

"Will not affect timestamp fields named explicitly by model options"